restangular/restangular.d.ts

```diff
@@ -90,11 +90,20 @@ declare namespace restangular {
     withConfig(configurer: (RestangularProvider: IProvider) => any): IService;
     restangularizeElement(parent: any, element: any, route: string, collection?: any, reqParams?: any): IElement;
     restangularizeCollection(parent: any, element: any, route: string): ICollection;
-    service(route: string, parent?: any): IService;
+    service(route: string, parent?: any): IScopedService;
     stripRestangular(element: any): any;
     extendModel(route: string, extender: (model: IElement) => any): void;
     extendCollection(route: string, extender: (collection: ICollection) => any): void;
   }
+  
+  interface IScopedService extends IService {
+    one(id: number): IElement;
+    one(id: string): IElement;
+    post(elementToPost: any, queryParams?: any, headers?: any): IPromise<any>;
```

Comment:
Restangular doesn't (and can't) guarantee that the promise returns the same type as `elementToPost`. But a server might make that guarantee, so I suppose the generic overload was added (in b57a5b10723c02624b877cdaf80d0011f54db1a4) as a convenience method for that case in lieu of a type assignment elsewhere.

Regardless, this PR's intention was to fix the behavior of `service` while preserving the semantics of the existing definition. If `post`'s generic overload is going to be removed, that should happen throughout the file, not just here in `IScopedService`.